game/static/game/css/lesson.css

```
@@ -171,51 +163,30 @@ h1 {
+    user-select:none;
+    font-size:clamp(22px,4vw,42px);
 }
 
 .lesson-board{
-    width:480px;
-    height:480px;
+    width:min(480px,90vw);
+    aspect-ratio:1;
+    height:auto;
 
     display:grid;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)